### PR TITLE
Fix dune 2.5 warning relative to menhir 2.1

### PR DESCRIPTION
**Pull request by emillon**
`(using menhir 2.1)` implies `dune >= 2.2.0` (see #1330).
Starting with dune 2.5.0, this triggers a warning.
Using `(lang dune 2.2)` in the main project ensures consistent metadata.

### CODE REVIEW

**Comment by emillon** [comment]
This is the innocuous part; the actual port is done in #1196.

File: dune [hunk 1]
```diff
@@ -21,8 +21,8 @@
    %{targets}
    (run ocamlformat --help=plain))))
 
-(alias
- (name runtest)
+(rule
+ (alias runtest)
```
Comment (Julow) [inline dune:25]: Is this required by the upgrade to 2.2 ?

Comment (emillon) [reply dune:25]: yes, `(alias)` does not exist anymore in `(lang dune 2.2)`.

Comment (Julow) [reply dune:25]: It is used in `test/passing/dune.inc` (of the same form) and the tests are running fine. Why ?

Comment (Julow) [reply dune:25]: I missed `test/passing/dune-project`. Looks good

**Comment by emillon** [comment]
Thanks

**Pull request merged**